### PR TITLE
Upgrade to Lerna 8

### PR DESCRIPTION
**Pull request by mfedderly**
Can someone double check the changes from running `lerna repair`?

https://github.com/lerna/lerna/blob/main/CHANGELOG.md#breaking-changes

### CODE REVIEW

File: lerna.json [hunk 1]
```diff
@@ -5,5 +5,6 @@
     "publish": {
       "registry": "https://registry.npmjs.org"
     }
-  }
+  },
+  "$schema": "node_modules/lerna/schemas/lerna-schema.json"
```
Comment (RobinVdBroeck) [inline lerna.json:9]: This is a nitpick, but normally a "$schema" is the first property in a JSON file. This makes it more discoverable when reading the file without any context.

**Review by RobinVdBroeck — APPROVED** [review]
LGTM

**Review by twelch — APPROVED** [review]
@mfedderly can/did you confirm that caching is still working for the 3 tasks as part of the top-level `prepare` and `test` scripts?

**Comment by mfedderly** [comment]
Yes they appear to still cache as expected.

**Pull request merged**